lux/core/frame.py

```diff
@@ -675,11 +722,15 @@ def render_widget(self, renderer: str = "altair", input_current_vis=""):
         import luxwidget
 
         widgetJSON = self.to_JSON(self._rec_info, input_current_vis=input_current_vis)
+        if pandasHtml is None:
```

Comment:
This is for the purpose of backwards compatibility with other versions of lux-api right?

Reply:
Perhaps - there was an error in one of the tests that caused pandasHtml to be none, so I just put a null check here

Reply:
Yeah, when we change the input to luxwidget, the missing traitlet variable can cause an error. So It's good that we have this check in here.